.github/workflows/maven-release.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will build maven releases
+
+name: Maven version release
+
+on:
+  push:
+    branches: release
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Release
+        uses: qcastel/github-actions-maven-release@master
```

Comment:
We should use a tagged version, instead of @master. The latest tagged version is fine for now. Maybe later, we'll just roll with @master, but I lack that confidence for now.